js/column.js

```diff
@@ -1663,6 +1663,15 @@ function ForeignKeyPseudoColumn (reference, fk, sourceObjectWrapper, name) {
 // extend the prototype
 module._extends(ForeignKeyPseudoColumn, ReferenceColumn);
 
+/**
+ * Given the available tuple data, generate the uniqueId for the selected row from the table this pseudo column points to
+ *
+ * @param {*} linkedData
```

Comment:
Can you please add a small description to this param that clarifies what this is. It's the key-value pair for the column values of the table that this pseudo column points to. I'm saying this because `linkedData` (Tuple.linkedData and what we're passing to `.filteredRef`) is not just a key-value pair of one table. It has the data for all the outbound fks.